### PR TITLE
Replace usage of array_get

### PR DESCRIPTION
**Pull request by BelleNottelling**
This PR replaces all uses of `array_get` with a ternary operator and completely removes it from the `di`.
Resulting code is cleaner, takes up less space, and is easier to read

I also fixed a case where the `bbmd` filter was still in use

### CODE REVIEW

**Comment by BelleNottelling** [comment]
Alright, I read through each and every file I changed for any mistakes, did some testing in both the admin panel and client area, and pushed some fixes. This should be ready for people to review and test for any potential issues

File: src/modules/Client/Api/Guest.php [hunk 1]
```diff
@@ -84,10 +84,10 @@ public function create($data = [])
         $this->getService()->checkExtraRequiredFields($data);
         $this->getService()->checkCustomFields($data);
 
-        $this->di['validator']->isPasswordStrong($this->di['array_get']($data, 'password'));
+        $this->di['validator']->isPasswordStrong($data['password']);
```
Comment (admdly) [inline src/modules/Client/Api/Guest.php:87]: Should `$data['password']` not default to `null` when no value is provided?

**Review by admdly — COMMENTED** [review]
Great PR - it's good to get these pointless legacy functions factored out. 😀 

Just one comment to double check where no longer defaulting to `null`, otherwise looks good to me.

File: src/modules/Client/Api/Guest.php (hunk 1 above)
Comment (BelleNottelling) [reply src/modules/Client/Api/Guest.php:87]: Nope, it should be fine without it. The password is one of the values that are marked as required, which is checked on [line 78](https://github.com/FOSSBilling/FOSSBilling/blob/main/src/modules/Client/Api/Guest.php#L78), so the function should never reach this point if it's not provided

**Review by admdly — APPROVED** [review]

**Review by John-S4 — APPROVED** [review]

**Pull request merged**